recipes/qt/5.x.x/conanfile.py

```diff
@@ -661,6 +661,15 @@ def package(self):
         # symbols that are also in "Qt5Core.lib". It looks like there is no "Qt5Bootstrap.dll".
         for fl in glob.glob(os.path.join(self.package_folder, "lib", "*Qt5Bootstrap*")):
             os.remove(fl)
+        tools.save(os.path.join("lib", "cmake", "Qt5Core", "extras.cmake"),
```

Comment:
```suggestion
        tools.save(os.path.join(self.package_folder, "lib", "cmake", "Qt5Core", "extras.cmake"),
```

Reply:
Maybe you want to store `os.path.join("lib", "cmake", "Qt5Core", "extras.cmake")` in a property since it's used in different blocks.